dwoo-plugins/log_report.php

```diff
@@ -2,7 +2,7 @@
 
 function Dwoo_Plugin_log_report(Dwoo_Core $dwoo, $groupers = array())
 {
-    if (!$GLOBALS['Session'] || !$GLOBALS['Session']->hasAccountLevel('Developer') || empty($_GET['log_report'])) {
+    if (!isset($GLOBALS['Session']) || !$GLOBALS['Session']->hasAccountLevel('Developer') || empty($_GET['log_report'])) {
```

Comment:
@nbey making a minor tweak here to use `empty()` instead of `!isset()`, it does some further checks that are probably good to have on guard. I prefer `empty` as the default choice for guard clauses